modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.server/src/main/scala/edu/gemini/seqexec/web/server/http4s/WebServerLauncher.scala

```diff
@@ -110,7 +113,7 @@ object WebServerLauncher extends ServerApp with LogInitialization {
   /**
     * Reads the configuration and launches the web server
     */
-  override def server(args: List[String]): Task[Server] = {
+  override def process(args: List[String]): Process[Task, Unit] = {
```

Comment:
Yes, that's exactly right. It was found that in many cases cleanup didn't proceed correctly. I don't think we do any cleanup but we might want to do it properly